js/src/main/scala/valueof/package.scala

```diff
@@ -0,0 +1,3 @@
+package object valueof {
+  @inline def valueOf[T](implicit vt: ValueOf[T]) = vt.value
```

Comment:
This is slightly hacky but it is basically the implementation of `scala.Predef.valueOf`